spec/requests/api/merge_requests_spec.rb

```diff
@@ -230,20 +233,20 @@
         expect(fork_project.forked?).to be_truthy
         expect(fork_project.forked_from_project).to eq(project)
         post api("/projects/#{fork_project.id}/merge_requests", user2),
-        title: 'Test merge_request', source_branch: "master", target_branch: "master", author: user2, target_project_id: project.id
+        title: 'Test merge_request', source_branch: 'master', target_branch: 'master', author: user2, target_project_id: project.id
         expect(response.status).to eq(201)
-        expect(json_response['title']).to eq('Test merge_request')
+        expect(json_response["title"]).to eq("Test merge_request")
       end
 
       it "should return 400 when source_branch is missing" do
         post api("/projects/#{fork_project.id}/merge_requests", user2),
-        title: 'Test merge_request', target_branch: "master", author: user2, target_project_id: project.id
+        title: 'Test merge_request', target_branch: 'master', author: user2, target_project_id: project.id
         expect(response.status).to eq(400)
       end
 
       it "should return 400 when target_branch is missing" do
         post api("/projects/#{fork_project.id}/merge_requests", user2),
-        title: 'Test merge_request', target_branch: "master", author: user2, target_project_id: project.id
+        title: 'Test merge_request', target_branch: 'master', author: user2, target_project_id: project.id
```

Comment:
Line is too long. [106/80]<br>Align the parameters of a method call if they span more than one line.